frontend/pages/ManageControlsPage/Scripts/components/DeleteScriptModal/DeleteScriptModal.tsx

```diff
@@ -44,7 +44,7 @@ const DeleteScriptModal = ({
         <p>
           The script{" "}
           <span className={`${baseClass}__script-name`}>{scriptName}</span> will
-          run on pending hosts. After the scripts runs, it&apos;s output and
+          run on pending hosts. After the scripts runs, its output and
```

Comment:
```suggestion
          run on pending hosts. After the script runs, its output and
```

Reply:
lol how did I miss that one? It was right next to it!